GitCommands/Settings/AppSettings.cs

```diff
@@ -1342,11 +1343,7 @@ public static bool ShowDiffForAllParents
             set => SetBool("showdiffforallparents", value);
         }
 
-        public static bool ShowSearchCommit
-        {
-            get => GetBool("showsearchcommit", false);
-            set => SetBool("showsearchcommit", value);
-        }
+        public static ISetting<bool> ShowSearchCommit { get; } = Setting.Create(DialogSettingsPath, nameof(ShowSearchCommit), false);
```

Comment:
This one is different.
```suggestion
        public static ISetting<bool> ShowSearchCommit { get; } = Setting.Create(AppearanceSettingsPath, nameof(ShowSearchCommit), false);
```